staging_service/autodetect/Mappings.py

```diff
@@ -29,6 +29,25 @@
 
 MEDIA = "KBaseBiochem.Media"
 
+fastq_reads_id = "fastq_reads"
```

Comment:
Is there documentation anywhere re how to add new mappings? It looks like it takes a json update as well as an update here?

Reply:
Added documentation. No, but I'll make an issue for it.